crates/bevy_pbr/src/render/mesh_view_bindings.wgsl

```diff
@@ -58,21 +58,23 @@ var dt_lut_texture: texture_3d<f32>;
 @group(0) @binding(15)
 var dt_lut_sampler: sampler;
 
+#ifdef PREPASS_TEXTURES
 #ifdef MULTISAMPLED
 @group(0) @binding(16)
 var depth_prepass_texture: texture_depth_multisampled_2d;
 @group(0) @binding(17)
 var normal_prepass_texture: texture_multisampled_2d<f32>;
 @group(0) @binding(18)
 var motion_vector_prepass_texture: texture_multisampled_2d<f32>;
-#else
+#else // MULTISAMPLED
 @group(0) @binding(16)
 var depth_prepass_texture: texture_depth_2d;
 @group(0) @binding(17)
 var normal_prepass_texture: texture_2d<f32>;
 @group(0) @binding(18)
 var motion_vector_prepass_texture: texture_2d<f32>;
```

Comment:
webgl shouldn't have an issue using textureLoad with the normal_prepass_texture & motion_vector_prepass_texture textures.